src/components/Dashboard/Notetaker/Category/CategorySidebar.jsx

```diff
@@ -83,11 +98,20 @@ const CategorySidebar = ({
                 <LoadingSpinner />
             ) : (
                 <>
+                    <div style={{ display: "flex", alignItems: "center", marginLeft: "10px" }}>
+                        {showDeleteAll && selectedCategories.length > 0 && (
+                            <div onClick={deleteAllCategories}>
+                                <BiTrash size="25px" fill="red" />
+                                <span style={{ marginRight: "5px", color: "orange" }}>Delete All</span>
+                            </div>
+                        )}
+                    </div>
                     <CategoryList
                         required
                         onPick={onPick}
                         pickedCategory={pickedCategory}
                         defaultCategory={defaultCategory}
+                        showCheckboxes={showDeleteAll}
```

Comment:
For the default category you don't need to showDelete button because this category can't be deleted